packages/optimizer/spec/transformers/valid/PreloadHeroImage/amp-img-object-fit-object-position/expected_output.html

```diff
@@ -0,0 +1,14 @@
+<html>
+<head>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <link rel="preload" href="https://amp.dev/static/img/icons/icon-512x512.png" as="image" data-hero>
+</head>
+<body>
+  <!-- object-fit and object-position should be rewritten to inline styles -->
+  <amp-img src="https://amp.dev/static/img/icons/icon-512x512.png" width="512" height="250" object-fit="cover" object-position="right top" data-hero i-amphtml-ssr><img class="i-amphtml-fill-content i-amphtml-replaced-content" decoding="async" src="https://amp.dev/static/img/icons/icon-512x512.png" styles="object-fit:cover;object-position:right top"></amp-img>
+  <!-- single attribute -->
+  <amp-img src="https://amp.dev/static/img/icons/icon-512x512.png" width="512" height="250" object-fit="cover" data-hero i-amphtml-ssr><img class="i-amphtml-fill-content i-amphtml-replaced-content" decoding="async" src="https://amp.dev/static/img/icons/icon-512x512.png" styles="object-fit:cover"></amp-img>
+  <!-- existing styles should not be passed through -->
+  <amp-img src="https://amp.dev/static/img/icons/icon-512x512.png" width="512" height="250" styles="background: red" object-fit="cover" object-position="right top" data-hero></amp-img>
```

Comment:
Since the third `amp-img` doesn't do SSR, should not the `style="background:red"` be put on the second `amp-img` to test how it behaves with SSR?

Reply:
good point. fixed